workflow-templates/upstream-sync.properties.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "StackHPC Sync With OpenStack Upstream",
+    "description": "Synchronise with the upstream OpenStack repository against specified releases"
```

Comment:
We could have this:

```
"filePatterns": [
        "^\.gitreview$"
    ]
```

Reply:
The filePatterns has been added. It complained about the backslash being an invalid escape character. Looking at the tox.properties.sync.json I noticed there is no backslash so I just used that instead.

Reply:
Without an escape, the regex will match agitreview, bgitreview, etc. I don't think it's an issue in practice though.